src/content/docs/full-stack-observability/observe-everything/get-started/get-started-auto-telemetry-pixie.mdx

```diff
@@ -0,0 +1,95 @@
+
+## Why it matters 
+
+Auto-telemetry with Pixie offers the quickest option for getting observability into your Kubernetes services. Did we mention no language agents are required?  
+
+Our Pixie integration gives you the best of both worlds: Pixie’s fast and simple Kubernetes observability coupled with New Relic One’s incident correlation, intelligent alerting, and long-term retention. You’ll get visibility into HTTP services using golden signals, HTTP transactions, database transactions, distributed tracing, and JVM metrics. You can operate, debug, and scale your Kubernetes clusters based on the information you learn about how your clusters and services are running. Using the New Relic [Explorer](/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts/), you can see key metrics and events at every level, starting with the cluster, and diving down into namespaces, deployments, and pods. You can quickly spot anomalous behavior, and where it’s happening. If you need further insight, you can easily link over to Pixie's native UI for advanced debugging with PXL scripts. 
+
+
+## Install Auto-telemetry with Pixie
+
+Use our [guided installation](/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview/) process to install Auto-telemetry with Pixie. This deploys Pixie with New Relic's Kubernetes integration on your cluster. You don't need to do any further configuration or installation to start using Pixie.
+
+If you want to install Auto-telemetry with Pixie on multiple clusters, re-run through the guided install for each additional cluster.
+
+### Beta limitations
+
+The following are not supported in this beta version of Auto-telemetry with Pixie:
+- OpenShift. While our New Relic infrastructure agent supports OpenShift, Pixie does not, so Auto-telemetry with Pixie cannot be used with OpenShift.
+- RBAC. New Relic RBAC controls cannot be used to gate access to Auto-telemetry with Pixie. This means the following:
+		• Auto-telemetry with Pixie creates one project for all of your Kubernetes clusters
+		• users from the same email domain are automatically allowed access to the project
+		• all users have full permissions on the project
+
+### Before you begin
+
+The guided install requires the following (you can get these set up before you begin, or during the install steps):
+
+- a [Pixie account](https://work.withpixie.ai/admin/api-keys)
+  You need a Google account to sign up for Pixie.
+- a Pixie API key 
+- Sufficient memory: Pixie requires 2Gb of memory per node.
+
+### Start the guided installation
+
+1. From [New Relic One](https://one.newrelic.com), click **Add more data**.
```

Comment:
You could lose a couple of steps by linking directly to the guided install (see also the Guided install overview doc buttons). But maybe you want to orient people to how to get there in the UI?

Reply:
This procedure has a LOT of steps! Is there a way to break it up somehow?

Reply:
Maybe you could have a "Prerequisite steps" procedure for people that need a Pixie account and API key and whatnot?